docs/quick-start/tips-and-tricks.md

```diff
@@ -4,23 +4,24 @@ title: Tips and tricks
 sidebar_label: Tips and tricks
 ---
 
-The following section gives you some quick ways to operate Titan. ``<n>`` means type a number on the keypad.
+The following section gives you some quick ways to operate Titan. ``<n>`` means type a number on the keypad. "\<Avo\> + " means hold down the first button.
 
 ## Fixture control and recording
 
--  Select all fixtures: \<AVO\>+ \<ALL\> \<ALL\> \<ALL\>
+-  Select all fixtures: \<AVO\> + \<ALL\> \<ALL\> \<ALL\>
 -  Add full intensity to the Programmer for selected fixtures: Tap \<@\>\<@\>
--  Add a percentage of intensity to the programmer: \<@\> + ``<n>`` \<Enter\> for percentage you want.
--  Clear over a time: ``<n>`` then \<Clear\> will clear over that time
+-  Add a percentage of intensity to the programmer: \<@\> ``<n>`` \<Enter\> for percentage you want.
```

Comment:
Oh I see, I misread this; I thought you were saying you could add 5% to 50 by doing `<@>[+]<10>` (which you can). Percentage can be a confusing term in that regard - maybe "Set a fixture to an intensity in the programmer:" would be clearer?

Reply:
I think it says percentage to imply a 2 digit entry - say on Eos if you did @ 5 enter you'd get 50% I think

Reply:
Ah yes. There is of course the user setting which affects this on Titan (https://manual.avolites.com/docs/en/system-settings/user-settings/#channel-levels)

Reply:
Perhaps maybe "Set a percentage intensity in the programmer:" instead? I think it's the word "add" which makes it ambiguous between;

1. set it to 110% of the existing value (not currently possible)
2. add 10 percentage points onto the current value or (use `[+]` softkey)
3. set it to 10% exactly (as described)

The first few times I read this I thought you meant either 1 or 2...!

Reply:
Ignore me, just saw you'd fixed it. lgtm